frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -118,124 +164,160 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
           onDownload={onDownloadAll}
         />
       </StackItem>
-      <StackItem>
-        <Split hasGutter isWrappable>
-          <SplitItem>
-            <Split hasGutter>
-              {isSmallScreen() ? null : (
-                <SplitItem>
-                  <Bullseye>
-                    <TextContent>
-                      <TextListItem component={TextListItemVariants.dt}>Step</TextListItem>
-                    </TextContent>
-                  </Bullseye>
-                </SplitItem>
-              )}
-              <SplitItem>
-                <SimpleDropdownSelect
-                  isDisabled={podContainers.length === 0}
-                  options={podContainers.map((container) => ({
-                    key: container.name,
-                    label: container.name,
-                  }))}
-                  value={selectedContainer?.name ?? ''}
-                  placeholder="Select container..."
-                  onChange={(v) => {
-                    setSelectedContainer(podContainers.find((c) => c.name === v) ?? null);
-                  }}
-                  width={150}
-                />
-              </SplitItem>
-            </Split>
-          </SplitItem>
-          <SplitItem>
-            <Button
-              variant={!logsLoaded ? 'plain' : isPaused ? 'plain' : 'link'}
-              onClick={() => setIsPaused(!isPaused)}
-              isDisabled={!!error}
-            >
-              {error ? (
-                'Error loading logs'
-              ) : !logsLoaded ? (
-                <>
-                  <Spinner size="sm" /> {isSmallScreen() ? 'Loading' : 'Loading log'}
-                </>
-              ) : isPaused ? (
-                <>
-                  <PlayIcon /> {isSmallScreen() ? 'Resume' : 'Resume refreshing'}
-                </>
-              ) : (
-                <>
-                  <PauseIcon /> {isSmallScreen() ? 'Pause' : 'Pause refreshing'}
-                </>
-              )}
-            </Button>
-          </SplitItem>
-          <SplitItem isFilled style={{ textAlign: 'right' }}>
-            {downloading ? (
-              <>
-                <Spinner size="sm" />{' '}
-              </>
-            ) : null}
-            {podContainers.length !== 0 ? (
-              <Dropdown
-                toggle={
-                  isSmallScreen() ? (
-                    <KebabToggle
-                      onToggle={() => setIsDownloadDropdownOpen(!isDownloadDropdownOpen)}
-                    />
-                  ) : (
-                    <DropdownToggle
-                      id="download-steps-logs-toggle"
-                      onToggle={() => setIsDownloadDropdownOpen(!isDownloadDropdownOpen)}
+      <StackItem isFilled id="complex-toolbar-demo">
+        <DashboardLogViewer
+          data={data}
+          logViewerRef={logViewerRef}
+          toolbar={
+            <Toolbar className="pf-v5-u-p-sm">
+              <ToolbarContent>
```

Comment:
Used Toolbar to avoid dark background of log viewer toolbar on full screen mode.
![Screenshot from 2023-11-28 14-54-51](https://github.com/opendatahub-io/odh-dashboard/assets/99261071/54e29b76-dcd6-444e-985f-49d012a07550)